cli/ops.rs

```diff
@@ -169,37 +168,30 @@ pub fn dispatch_all_legacy(
   }
 }
 
-pub fn op_selector_compiler(inner_type: msg::Any) -> Option<OpCreator> {
```

Comment:
What's your plan here? Add it back later?

Reply:
Yes. For various reasons, it's much easier for the compiler worker to share the state with the calling worker. I do like the ability to limit the ops to each worker - but this patch is quite complex and sharing the state is the easiest thing to get it working now.